modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -178,11 +179,21 @@ public abstract class Base${schemaName}ResourceImpl
 			<#elseif stringUtil.equals(javaMethodSignature.methodName, "get" + schemaName + "PermissionsPage")>
 				<#assign generateGetPermissionCheckerMethods = true />
 
+				Long resourceId = getPermissionCheckerResourceId(${schemaVarName}Id);
```

Comment:
I think that the order by use also applies here.

Reply:
Sorry, I was checking the code and reviewing the Source Formatter rules, but I don't know what are you referring to

Reply:
Variables have to be defined in the order they are used, it's a Brian's convention.

Reply:
I was thinking in alphabetical order here. I'll change them, thank you!